include/zephyr/bluetooth/conn.h

```diff
@@ -1329,6 +1451,67 @@ int bt_conn_le_subrate_set_defaults(const struct bt_conn_le_subrate_param *param
 int bt_conn_le_subrate_request(struct bt_conn *conn,
 			       const struct bt_conn_le_subrate_param *params);
 
+/** @brief Read Minimum Supported Connection Interval Groups.
+ *
+ *  Read the minimum supported connection interval and supported interval
+ *  groups from the local controller. This information describes what the
+ *  local controller supports.
+ *
+ *  @note To use this API @kconfig{CONFIG_BT_SHORTER_CONNECTION_INTERVALS} must be set.
+ *
+ *  @param info Pointer to structure to receive the minimum connection interval
+ *              information.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ */
+int bt_conn_le_read_min_conn_interval_groups(struct bt_conn_le_min_conn_interval_info *info);
```

Comment:
This struct this function takes as argument is 248 bytes (+1 byte alignment). This could make the API a unwieldly to use in constrained systems. Will every caller be interested in all the info? Maybe we should have a method for obtaining just the `min_supported_conn_interval`? And a way to read out the data in parts?

Reply:
Sorry, I meant to copy-paste `bt_conn_le_read_min_conn_interval` not `set_defaults`. The point about adding `@sa` in the documentation stands. What do you think about that?

Reply:
Sounds good